Sensor.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Sensor.h"
+
+uint32_t Sensor::getId() const { return id; }
+
+bool Sensor::isCritical() const { return criticality; }
+
+bool Sensor::healthCheck() const { return true; } // Unsure what best default is?
```

Comment:
I best default is no default imo. if any false. Its better to assume something is broken rather then working. We never have a raw Sensor Class and each type of sensor will check if its plugged in a unique way

Reply:
Fair enough. I'll make it pure virtual in a future PR.